### PR TITLE
Add chinese-dictionary extension

### PR DESCRIPTION
**Pull request by ilyaliao**
## Description

A Raycast extension that explains text using a Chinese dictionary.

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by mil3na** [comment]
Hi!
Thank you for your contribution! My name is Milena and I will be reviewing it today.

I have a few comments about your extension:
- We already have a few dictionary extensions at Raycast. A differential for yours would be the service you are using. Do you mind adding it to the extension name?
- Is there a way to create an account being outside of China? I didn't manage to create one to get an API key.
- The Motivation part on your README sounds a bit odd. Could you please rephrase it or remove it? I don't think we need motivation to use a dictionary.

**Comment by CofCat456** [comment]
Hi Milena,

Thank you for reviewing my pull request and providing your feedback.
Here are my responses to your comments:

1. I have updated the README to highlight the use of the Educational Encyclopedia in the description and features section to better differentiate my extension.
2. 
	Users can follow the link in the README to the login page of the Educational Encyclopedia, they can sign in using a 
Google or Apple account.
	Additionally, they can obtain a test API key from the [Educational Encyclopedia’s Swagger page](https://pedia.cloud.edu.tw/apiui.html#/%E8%A9%9E%E6%A2%9D/get_List).
3.	I have removed the Motivation part from the README and added more detailed usage instructions.

Try entering **樂不思蜀** and you will get the meaning of the term along with an example sentence.

Thank you again for your valuable feedback.

**Comment by mil3na** [comment]
Hey!
Thanks for the instructions, I managed to use it now!
But, I'm missing the changes on the README on this PR.
Also, on the first state, the default action is to open Github. Is that expected? If so, why?

File: extensions/chinese-dictionary/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "chinese-dictionary",
+  "title": "Chinese Dictionary",
+  "description": "Chinese Dictionary",
```
Comment (LitoMore) [inline extensions/chinese-dictionary/package.json:5]: How about changing the name to `Mandarin Chinese Dictionary`? It's more accurate. You can also catch users who use `mandarin` as a search keyword.

File: extensions/chinese-dictionary/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  "author": "CofCat",
+  "categories": [
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "chinese-dictionary",
+      "title": "Chinese Dictionary",
+      "description": "Find the textual definition",
```
Comment (LitoMore) [inline extensions/chinese-dictionary/package.json:16]: You could use `Find Textual Definition` as the title. Your command name is a duplicate of the extension name here. Using a more specific name allows users to reach our command faster.

File: extensions/chinese-dictionary/package.json (hunk 1 above)
Comment (CofCat456) [reply extensions/chinese-dictionary/package.json:5]: That’s a great idea!

Thanks!

File: extensions/chinese-dictionary/package.json (hunk 2 above)
Comment (CofCat456) [reply extensions/chinese-dictionary/package.json:16]: Thanks for the suggestion!

**Comment by CofCat456** [comment]
Hi @mil3na,

Sorry, it took me a while to figure out how to update my local commit to here.

I already updated the README.

Thanks to @LitoMore suggestion, I renamed the extension, but this led to the creation of a [new PR](https://github.com/raycast/extensions/pull/13309). Should I close this current one?

**Comment by LitoMore** [comment]
@CofCat456 Yes. Let's move to that PR and close this one.

**Pull request closed without merging**